Makefile

```diff
@@ -1,5 +1,8 @@
 # Dummy makefile, will call the host and enclave makefile when requested.
 
+C?= clang
+CXX?= clang++
```

Comment:
I know we use clang or gcc as compiler but why do these variables need to be set? 

Is there a typo here and can we add a space before the `?=` for readability :) like so:
```
CC ?= clang
CXX ?= clang++
```

Reply:
By default the build calls `cc` when compiling the C code but the `clang` package, unlike the `gcc` package, does not create a symbolic link from `cc` to the compiler.  Similarly for C++.  Hence,  either the environment needs to specify the relevant compilers or the `cc` and `c++` symbolic links have to be added some where in $PATH

You are correct, that is a typo.  Nice catch!  Also, I agree regarding the spacing.  I will make that change as well.